Dockerfile.build

```diff
@@ -0,0 +1,5 @@
+FROM ubuntu:16.04
+RUN apt-get update
+RUN apt-get install -y golang git-core libgpgme11-dev
+ENV GOPATH=/
+WORKDIR /src/github.com/projectatomic/skopeo
```

Comment:
ATM I think the development happens on Fedora.  OTOH having the build Dockerfile use Ubuntu and the test one use Fedora/CentOS might be good for interoperability :)